internal/scaffold/entrypoint_test.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/operator-framework/operator-sdk/internal/util/diffutil"
 )
 
-func EntrypointTest(t *testing.T) {
+func Test_EntrypointTest(t *testing.T) {
```

Comment:
I've never seen underscores (_) used in our code. I would call it `TestEntrypoint` instead.